packages/nimble-components/src/anchor-menu-item/tests/anchor-menu-item.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { html } from '@microsoft/fast-element';
+import { AnchorMenuItem } from '..';
+import { waitForUpdatesAsync } from '../../testing/async-helpers';
+import { fixture, Fixture } from '../../utilities/tests/fixture';
+import { getSpecTypeByNamedList } from '../../utilities/tests/parameterized';
+
+async function setup(): Promise<Fixture<AnchorMenuItem>> {
+    return fixture<AnchorMenuItem>(
+        html`<nimble-anchor-menu-item href="#"></nimble-anchor-menu-item>`
+    );
+}
+
+describe('Anchor Menu Item', () => {
```

Comment:
Is it possible to write unit tests that verify our FAST workarounds in `index.ts` are working?

Reply:
I was able to write one for the `instanceof` override, but I couldn't get a test to work for the other workarounds. Specifically, focusing an item and then dispatching keyboard events to arrow through the menu didn't work. Focus never moved from the initially focused item. Additionally, I don't know how to test that clicking or pressing Enter actually navigates. Seems like it would need to be a playwright test or something.